demisto_sdk/commands/common/tests/release_notes_test.py

```diff
@@ -475,3 +476,48 @@ def test_get_categories_from_rn(method_to_check, release_notes_content, filled_e
     - Case 3: Should Create a dict with two keys of integraition 1 and integration 2.
     """
     assert method_to_check(ReleaseNotesValidator, release_notes_content) == filled_expected_result
+
+
+def update_json(path, key, value):
+    import json
+
+    js = get_dict_from_file(path=path)[0]
+    js[key] = value
+    with open(path, 'w') as f:
+        json.dump(js, f)
+
+
+TEST_RELEASE_NOTES_BREAKING_CHANGE = [
+    ("\n#### Integrations\n##### Integration name\n" +
+     "- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.\n", False, False, True),
+    ("\n#### Integrations\n##### Integration name\n" +
+     "- Breaking change test\n", False, False, False),
+    ("\n#### Integrations\n##### Integration name\n" +
+     "- Breaking change test\n", True, True, False),
+    ("\n#### Integrations\n##### Integration name\n" +
+     "- Breaking change test\n", True, False, True)
+]
+
+
+@pytest.mark.parametrize('release_notes_content, has_json, change_json, expected_result', TEST_RELEASE_NOTES_BREAKING_CHANGE)
+def test_is_breaking_change(release_notes_content, has_json, change_json, expected_result, mocker, repo):
+    """
+    Given
+    - A release note.
+    When
+    - Run is_breaking_change validation.
+    Then
+    - Ensure that if the release note contains 'breaking change', there is also an appropriate json file.
+    """
+    mocker.patch.object(ReleaseNotesValidator, '__init__', lambda a, b: None)
```

Comment:
if we mock this, what will be the value of `self.latest_release_notes` in every test case?

Reply:
The ***get_validator*** method will set the `self.latest_release_notes` with the `release_notes_content` argument for each test case.